quiz/system_tests/test_contribute.py

```diff
@@ -0,0 +1,57 @@
+from django.conf import settings
+from django.core import mail
+
+from quiz.system_tests.system_test_case import SystemTestCase
+
+
+class ContributingQuestionsTest(SystemTestCase):
+    def test_user_contributes_a_valid_question(self):
+        self.create_a_question()
+        self.should_see_the_contribute_form()
+        self.fill_in_the_spam_protection_correctly()
+        self.fill_in_the_question_and_explanation('foo', 'bar')
+        self.the_administrators_should_get_an_email_about_a_new_question()
+
+    def test_user_forgets_to_enter_the_question_itself(self):
+        self.create_a_question()
+        self.should_see_the_contribute_form()
+        self.fill_in_the_spam_protection_correctly()
+        self.fill_in_the_question_and_explanation('', 'bar')
+        self.there_is_an_error('This field can not be empty.', 'question')
+
+    def test_user_forgets_to_enter_an_explanation(self):
+        self.create_a_question()
+        self.should_see_the_contribute_form()
+        self.fill_in_the_spam_protection_correctly()
+        self.fill_in_the_question_and_explanation('foo', '')
+        self.there_is_an_error('This field can not be empty.', 'explanation')
+
+    def test_user_contributes_a_valid_question_but_fails_spam_protection(self):
+        self.create_a_question()
+        self.should_see_the_contribute_form()
+        self.fill_in_the_spam_protection_incorrectly()
+        self.fill_in_the_question_and_explanation('foo', 'bar')
+        self.there_is_an_error('You failed the spam protection', 'spam_protection')
+
+    def create_a_question(self):
```

Comment:
Maybe rename to `visit_contribution_page` or something? Since this function doesn't actually create a question.